http/ubuntu/preseed-hyperv.cfg

```diff
@@ -0,0 +1,34 @@
+choose-mirror-bin mirror/http/proxy string
+d-i base-installer/kernel/override-image string linux-server
+d-i clock-setup/utc boolean true
+d-i clock-setup/utc-auto boolean true
+d-i finish-install/reboot_in_progress note
+d-i grub-installer/only_debian boolean true
+d-i grub-installer/with_other_os boolean true
```

Comment:
The changes added to the main Ubuntu preseed file in #838 are missing from this hyper-v variant, would you mind opening a PR to add them in? :-)